fdbclient/NativeAPI.actor.cpp

```diff
@@ -169,6 +169,34 @@ void DatabaseContext::removeTssMapping(StorageServerInterface const& ssi) {
 	}
 }
 
+void DatabaseContext::addSSIdTagMapping(const UID& uid, const Tag& tag) {
+	ssidTagMapping[uid] = tag;
+}
+
+void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& locationInfo,
```

Comment:
add comments describing the function

Reply:
> add comments describing the function (here, elsewhere)
Done already. The comments are in DatabaseContext.h. I can copy them here if you think that would be helpful.

> useful: `TraceEvent`s (commented out, but available for debugging)